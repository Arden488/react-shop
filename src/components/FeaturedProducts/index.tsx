import * as React from 'react';

import './styles.css';

const FeaturedProducts = () => (
  <div>
    FeaturedProducts
  </div>
);

export default FeaturedProducts;
