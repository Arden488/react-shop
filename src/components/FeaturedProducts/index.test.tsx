import { shallow } from 'enzyme';
import * as React from 'react';

import FeaturedProducts from './index';

let component: React.ReactNode = null;

beforeEach(() => {
  component = shallow(<FeaturedProducts />);
});

it('renders correctly', () => {
  expect(component).toMatchSnapshot();
});
