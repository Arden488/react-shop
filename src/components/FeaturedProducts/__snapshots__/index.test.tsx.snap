// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="product-list product-list_featured"
>
  <CatalogElementListItem
    code="1"
    key="1"
  />
  <CatalogElementListItem
    code="1"
    key="2"
  />
  <CatalogElementListItem
    code="1"
    key="3"
  />
</div>
`;
